src/score_hv/harvesters/inc_logs.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+
+""" Collection of methods to faciliate retrieval of increment
+    descriptive statistics from logs
+"""
+
+import os
+
+from collections import namedtuple
+from dataclasses import dataclass
+from dataclasses import field
+
+from score_hv.config_base import ConfigInterface
+
+HRVSTER_NAME = 'inc_logs'
```

Comment:
is this spelled without an a for a reason? may be better spelled out for readability

Reply:
for consistency with innov_netcdf.py; corrected in both to "HARVESTER_NAME"